src/fragments/lib-v1/datastore/native_common/relational.mdx

```diff
@@ -130,6 +130,12 @@ However, in a many-to-many relationship the children are not removed and you mus
 
 For many-to-many relationships, you can use the `@manyToMany` directive and specify a `relationName`. Under the hood, Amplify creates a join table and a one-to-many relationship from both models.
 
+<Callout warning>
```

Comment:
Similar to above, I believe this is correct and is present in vCurrent but would suggest removing it to limit scope in this large PR.